docs/src/main/sphinx/sql/show-functions.rst

```diff
@@ -12,7 +12,8 @@ Synopsis
 Description
 -----------
 
-List all the functions available for use in queries.
+List all the functions available for use in queries in a SQL client connected
+to Trino.
```

Comment:
Should mention that built-in and custom functions are supported and maybe have an example query with example results as above.